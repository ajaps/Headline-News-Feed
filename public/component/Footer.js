import React from 'react';

export default class Footer extends React.Component {
  render() {
    return (
    <footer>Headline News App (c) 2017 </footer>
    );
  }
}
