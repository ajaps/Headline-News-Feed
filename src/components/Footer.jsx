import React from 'react';

const Footer = () => (
    <p className="footer">Headline RSS Feed App (c) 2017 </p>
  );

export default Footer;
