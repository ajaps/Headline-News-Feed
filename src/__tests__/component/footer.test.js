import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import Footer from '../../components/Footer.jsx';


describe('Footer component', () => {
  let app;
  beforeEach(() => {
    app = shallow(<Footer />);
  });

  it('should render as expected', () => {
    const tree = toJson(app);
    expect(tree).toMatchSnapshot();
  });

  it('should contain name of the app "Healine News" ', () => {
    expect(app.exists('Headline RSS Feed App (c) 2017 ')).toBe(true);
  });
});
