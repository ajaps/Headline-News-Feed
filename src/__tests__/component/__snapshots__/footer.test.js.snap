// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render as expected 1`] = `
<p
  className="footer"
>
  Headline RSS Feed App (c) 2017 
</p>
`;
